src/OpenTelemetry.Abstractions/Trace/Tracing.cs

```diff
@@ -0,0 +1,81 @@
+﻿// <copyright file="Tracing.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Trace
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using OpenTelemetry.Context;
+    using OpenTelemetry.Context.Propagation;
+    using OpenTelemetry.Trace.Export;
+
+    /// <summary>
+    /// Class that manages a global instance of the <see cref="Tracer"/>.
+    /// </summary>
+    public static class Tracing
+    {
+        private static readonly Proxy ProxyInstance = new Proxy();
+
+        /// <summary>
+        /// Gets the tracer to record spans.
+        /// </summary>
+        public static ITracer Tracer { get; } = ProxyInstance;
+
+        /// <summary>
+        /// Gets the exporter to use to upload spans.
+        /// </summary>
+        public static ISpanExporter SpanExporter { get; } = ProxyInstance;
+
+        public static void Init(ITracer tracer, ISpanExporter spanExporter)
+        {
+            ProxyInstance.ActualTracer = tracer ?? NoopTracer.Instance;
```

Comment:
this is prone to race-conditions: what if tracer has changed but exporter did not yet?

Reply:
I'm aware. I'd use `Interlocked` here to make sure Init is called once. The spec proposal you sent said it should be logged if the second attempt comes in and not allow swap. I've made a comment below wrt that.

Reply:
Oh I see what you mean.
Does that mean `Tracer` and `Exporter` need to set atomically? I assumed if the Tracer is used with the `NoopExporter` before the actual exporter reference is assigned, it shouldn't be an issue.
Would you prefer setting both properties to be atomic?

Reply:
check out som changes I'm working on: https://github.com/open-telemetry/opentelemetry-dotnet/blob/3751a863b9ca0ccac0f6d81f83fa81832347afad/src/OpenTelemetry/Trace/Tracing.cs

Tracer will be the only default thing we will set up, tracer will know about underlying exporter.. 

`Tracing` or `DefaultTracing` don't need to know about exporter at all.